ad-insertion/video-analytics-service/app/modules/PipelineManager.py

```diff
@@ -135,7 +135,7 @@ def stop_instance(name, version, instance_id):
     @staticmethod
     def is_pipeline_exists(name, version, instance_id=None):
         if name not in PipelineManager.pipelines or \
-                version not in PipelineManager.pipelines[name]:
+                str(version) not in PipelineManager.pipelines[name]:
```

Comment:
can we store version in map as integer?

Reply:
I think we should but I just found this issue, strangely it works sometime and doesn't at some places

Reply:
I mean integer as key works at some places in code without any issues

Reply:
let's put a task / item to clean that up - need to make it consistent